indico/modules/events/abstracts/templates/abstract/public.html

```diff
@@ -479,4 +497,7 @@ <h3 class="abstract-title">
             </a>
         {% endif %}
     </div>
+    <script>
+        $('.dropdown-abstract-pdf').parent().dropdown();
```

Comment:
I think we have some declarative way of making the dropdown elements automatically initialized.